Makefile

```diff
@@ -106,7 +107,11 @@ test-coverage: generate
 
 .PHONY: perf
 perf: generate
-	$(GO) test $(GO_TAGS) -run=- -bench=. -benchmem ./...
+	$(GO) test $(GO_TAGS),slow $(GO_TEST_TIMEOUT) -run=- -bench=. -benchmem ./...
+
+.PHONY: wasm-sdk-e2e-test
+wasm-sdk-e2e-test: generate
+	$(GO) test $(GO_TAGS),slow,wasm_sdk_e2e $(GO_TEST_TIMEOUT) ./internal/wasm/sdk/test/e2e
```

Comment:
:mag: Looking at [the output for this PR](https://github.com/open-policy-agent/opa/pull/2929/checks?check_run_id=1443959635#step:3:158), I'd appreciate `-v` here. Just to have some more action in the logs 😄

Reply:
Fixed